locale/circuitpython.pot

```diff
@@ -1411,6 +1411,10 @@ msgstr ""
 msgid "Pull not used when direction is output."
 msgstr ""
 
+#: ports/stm/ref/pulseout-pre-timeralloc.c
+msgid "PulseOut not supported on this chip"
```

Comment:
Is this from a separate change?

Reply:
I actually have no idea where this came from. It just showed up out of nowhere when I ran make-translate?

Reply:
It may have been left over from an old commit, and I just didn't run Make Translate recently enough?

Reply:
Do you have a `ports/stm/ref` directory outside of git? Looks like it found the string there.

Reply:
Oh you're right, I missed that. That's a gitignored folder I use to hold the stm32 micropython directory for porting reference. I wonder why it only started finding it now?